psec/pinblock.py

```diff
@@ -164,6 +171,157 @@ def encode_pinblock_iso_3(pin: str, pan: str) -> bytes:
     return _tools.xor(pinblock, pan_block)
 
 
+def encode_pin_field_iso_4(pin: str, det_pad: Optional[bytes] = None) -> bytes:
+    r"""Encode ISO 9564 PIN block format 4 plain text PIN field.
+    ISO format 4 PIN plain text PIN field is a 16 byte value that consits of
+
+        - Control field. A 4 bit hex value set to 4.
+        - PIN length. A 4 bit hex value in the range from 4 to C.
+        - PIN digits. Each digit is a 4 bit hex value in the range from 0 to 9.
+        - Fill digits. Each digit is a 4 bit hex value set to A.
+        - Random pad character. A 4 bit hex value in the range from 0 to F.
+
+    Parameters
+    ----------
+    pin :   str
+            ASCII Personal Identification Number.
+    det_pad (opt) : bytes
+                    Optional binary 8-byte deterministic padding to be used instead of random padding
+                    supporting deterministic test cases.
```

Comment:
I don't have it mention anywhere, but the package uses numpy doc style. https://numpydoc.readthedocs.io/en/latest/format.html

So, this block should be https://numpydoc.readthedocs.io/en/latest/format.html#parameters:

```
    Parameters
    ----------
    pin : str
        ASCII Personal Identification Number.
    det_pad : bytes, optional
        Optional binary 8-byte deterministic padding to be used instead of random padding
        supporting deterministic test cases.
```

Reply:
Thanks for the information.